docs/docs/FAQ.mdx

```diff
@@ -387,4 +387,20 @@ If your version of Immich is below 1.92.0 and the crash occurs after logs about
 
 Sometimes Immich logs errors such as "duplicate key value violates unique constraint" or "column (...) of relation (...) already exists". Because of Immich's container structure, this error can be seen when both immich and immich-microservices start at the same time and attempt to migrate or create the database structure. Since the database migration is run sequentially and inside of transactions, this error message does not cause harm to your installation of Immich and can safely be ignored. If needed, you can manually restart Immich by running `docker restart immich immich-microservices`.
 
+### Why does foreground backup stop when I navigate away from the app? Shouldn't it transfer the job to background backup?
+
+Foreground backup and background backup are two separate mechanisms. They don't communicate or interact with each other.
+
+Foreground backup is controlled by the user's action, while background backup is controlled by your phone's operating system. When the app is put in the background, the invocation of background tasks is delegated to the phone's operating system scheduler. It makes the call when the background task can run and how long it can run.
```

Comment:
```suggestion
Foreground backup is controlled by the user's action, while background backup is controlled by your device's operating system. When the app is put in the background, the invocation of background tasks is delegated to the device's operating system scheduler. It makes the call when the background task can run and how long it can run.
```